.vscode/launch.json

```diff
@@ -268,6 +268,24 @@
 				"group": "2_attach"
 			}
 		},
+		{
+			/* Added for "VS Code Selfhost Test Provider" extension support */
+			"type": "pwa-chrome",
+			"request": "attach",
+			"name": "Attach to VS Code",
+			"browserAttachLocation": "workspace",
+			"port": 9222,
+			"outFiles": [
+				"${workspaceFolder}/out/**/*.js"
+			],
+			"presentation": {
+				"group": "2_attach"
```

Comment:
Could you try setting "hidden" to true here though and see if it still runs? It'd be nice not to have this show up in the drop down - since it'll only be used by the selfhost test provider extension.